### PR TITLE
Added the changes as suggested under issue no.#76

### PR DESCRIPTION
**Pull request by abhay-raj19**
## Screenshots
image

## Issue ticket number and link →
 >https://github.com/devarshishimpi/stonecss/issues/76

## Checklist before requesting a review

-✅I have performed a self-review of my code.

-✅Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).

-✅I ran the app and tested it locally to verify that it works as expected.

-✅I have starred the repository.

## Additional Information

 >🧿 I Had Added the Changes As Suggested under #76 , If  (@devarshishimpi)  you do like it pls merge it.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -9,6 +9,38 @@ body {
...
+    color: #FFFFFF;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    z-index: 1;
+}
+
+#btnScrollToTop:hover {
+    width: 52px;
+    height: 52px;
```
Comment (MrKrishnaAgarwal) [inline style.css:32]:
```suggestion
    width: 50px;
    height: 50px;
```
I suggest not to increase the height and width here or keep it same above too.
else it doesn't looks good

File: style.css [hunk 2]
```diff
@@ -9,6 +9,38 @@ body {
...
 }
 
+/* scroll-up button starts */
+
+#btnScrollToTop {
+    position: fixed;
+    right: 10px;
+    bottom: 10px;
+    width: 50px;
+    height: 50px;
```
Comment (MrKrishnaAgarwal) [inline style.css:19]: you should keep the same height and width as mentioned above

**Review by MrKrishnaAgarwal — CHANGES_REQUESTED** [review]
please resolve below comments

**Review by MrKrishnaAgarwal — APPROVED** [review]
Looks good!
Thanks for contributing @abhay-raj19 🎉

**Review by devarshishimpi — APPROVED** [review]
Thanks For Contributing!

File: style.css (hunk 1 above)
Comment (abhay-raj19) [reply style.css]: Thankyou😊.( @MrKrishnaAgarwal)

**Comment by MrKrishnaAgarwal** [comment]
@devarshishimpi please check #78

**Comment by devarshishimpi** [comment]
> @devarshishimpi please check #78

Done

**Comment by MrKrishnaAgarwal** [comment]
> > @devarshishimpi please check #78
> 
> Done

Thanks!!

**Pull request merged**